commands/configure.go

```diff
@@ -99,13 +105,22 @@ func loadOrInitializeEnvironment(location string) (themekit.Environments, error)
 	}
 
 	env, err := themekit.LoadEnvironments(contents)
-	if (err != nil && canProcessWithError(err)) || len(env) <= 0 {
+
+	if err != nil && !canProcessWithError(err) {
+		return env, err
+	}
+
+	if err != nil || len(env) <= 0 {
 		conf, _ := themekit.LoadConfiguration(contents)
 		env[themekit.DefaultEnvironment] = conf
 	}
 	return env, err
 }
 
 func canProcessWithError(e error) bool {
-	return strings.Contains(e.Error(), "YAML error") == false
+	if strings.Contains(e.Error(), "YAML error") == false {
```

Comment:
Hmmm, I guess the yaml parser doesn't expose custom error types? This seems brittle.